src/service/modes.js

```diff
@@ -0,0 +1,2 @@
+export const WHO_IS_THAT_POKEMON = 'WHO_IS_THAT_POKEMON';
+export const WHAT_DOES_THIS_POKEMON_LOOK_LIKE = 'WHAT_DOES_THIS_POKEMON_LOOK_LIKE';
```

Comment:
Can we do something like this for each mode? I think it could be useful for quiz page rendering, and we'll keep all information about modes in one file.
```
WHO_IS_THAT_POKEMON = {
        name: 'WHO_IS_THAT_POKEMON',
        title: "Who's that pokemon?",
        questionType: "image",
        answerType: "text",
        answersNumber: 4
    }
```
You only need to change 
`this.mode = mode;` to `this.mode.name`

Reply:
Cool :) I like the idea! Let's do this @mariusz-sm

Reply:
I think that it is good idea. @lukaszdutka What do you think? Can I change it in above mentioned way?

Reply:
Done :) I added new type of mode.